### PR TITLE
Added ScaLarList comp to fix properties with List settings not displayed issue 

### PR DESCRIPTION
**Pull request by 4manasa**

### CODE REVIEW

File: packages/react-sdk-components/src/components/field/ScalarList/ScalarList.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import FieldValueList from '../../designSystemExtension/FieldValueList';
+import PropTypes from 'prop-types';
```
Comment (tumms2021389) [inline packages/react-sdk-components/src/components/field/ScalarList/ScalarList.tsx:4]: move this import one line up

File: packages/react-sdk-components/src/components/designSystemExtension/FieldValueList/FieldValueList.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import { makeStyles } from '@material-ui/core/styles';
 interface FieldValueListProps{
   // If any, enter additional props that only exist on this component
   name?: string,
-  value: string,
+  value: any,
```
Comment (tumms2021389) [inline packages/react-sdk-components/src/components/designSystemExtension/FieldValueList/FieldValueList.tsx:12]: can't you define exact types it accept?

Comment (4manasa) [reply packages/react-sdk-components/src/components/designSystemExtension/FieldValueList/FieldValueList.tsx:12]: The value can be rect element, string or number format

File: packages/react-sdk-components/src/components/field/ScalarList/ScalarList.tsx (hunk 1 above)
Comment (4manasa) [reply packages/react-sdk-components/src/components/field/ScalarList/ScalarList.tsx:3]: Done

**Review by JEAxero — APPROVED** [review]
Reviewed. Tested. Looks good to me.

**Pull request merged**